docroot/modules/custom/mass_metatag/tests/src/ExistingSite/LocationSpecialAnnouncementMetadataTest.php

```diff
@@ -104,7 +104,7 @@ public function getExpectedMetatags(ContentEntityInterface $entity) {
       'geo.placename' => $entity->label(),
       'geo.region' => 'US-MA',
       'mg_phone_number' => '123-456-7890',
-      'mg_address' => "123 Test Way\nBoston, MA 12345\nUnited States",
+      'mg_address' => "123 Test Way Boston, MA 12345 United States",
```

Comment:
@arthurbaghdas I'm curious why the new lines were removed here. Is this a scenario we don't need to think about with the schema_metatag changes?